ebuild-writing/variables/text.xml

```diff
@@ -276,6 +276,15 @@ The following variables may or must be defined by every ebuild.
     <uri link="::ebuild-writing/eapi/#eapi=4">EAPI>=4</uri>.)
     </ti>
   </tr>
+  <tr>
+    <ti><c>HTML_DOCS</c></ti>
+    <ti>
+    An array or space-delimited list of documentation files for
+    the default src_install function to install using dodoc into
```

Comment:
It's technically the `einstalldocs` function, not `src_install`. the fact that `src_install` calls `einstalldocs` is irrelevant.

We should mention that it's done recursively per `dodoc -r`.

I also find the text a bit hard to follow. I'm not sure if the developers need to care where the files are installed, they just need to know it's for the html documentation files.